qutebrowser/javascript/caret.js

```diff
@@ -932,7 +945,7 @@ window._qutebrowser.caret = (function() {
      */
     CaretBrowsing.setCaretElementNormalStyle = function() {
         const element = CaretBrowsing.caretElement;
-        element.className = "CaretBrowsing_Caret";
+        element.ClassName = "CaretBrowsing_Caret";
```

Comment:
Why this change? `className` seems to be the canonical spelling: https://developer.mozilla.org/en-US/docs/Web/API/Element/className